tools/src/bin/derive-per-file-info.rs

```diff
@@ -208,7 +208,7 @@ fn traverse_and_store_coverage(cov_node: &mut json::Object, path_so_far: &str, d
 ///         `if (os == "win"): https://bugzilla.mozilla.org/show_bug.cgi?id=NNN`
 ///         under a "disabled" mochitest ini-format header would result in
 ///         `["os == \"win\"\n", "https://bugzilla.mozilla.org/show_bug.cgi?id=NNN"]
-///         and a line line the following directly under the test name
+///         and a line like the following directly under the test name
```

Comment:
I had to ask someone for help, because, again, my default is to just drop duplicate words, but that made no sense.

Reply:
Thanks for going the extra mile on this one!  This does indeed seem to be what the intent was!